Makefile

```diff
@@ -1,13 +1,143 @@
+SHELL              := /bin/bash
+# go options
 GO                 ?= go
 LDFLAGS            :=
 GOFLAGS            :=
 BINDIR             ?= $(CURDIR)/bin
+GO_FILES           := $(shell find . -type d -name '.cache' -prune -o -type f -name '*.go' -print)
+GOPATH             ?= $$($(GO) env GOPATH)
+DOCKER_CACHE       := $(CURDIR)/.cache
 GO_VERSION         := $(shell head -n 1 build/images/deps/go-version)
 
 DOCKER_BUILD_ARGS = --build-arg GO_VERSION=$(GO_VERSION)
 
+.PHONY: all
+all: build
+
 include versioning.mk
 
+UNAME_S := $(shell uname -s)
+
+.PHONY: bin
+bin:
+	@mkdir -p $(BINDIR)
+	GOOS=linux $(GO) build -o $(BINDIR) $(GOFLAGS) -ldflags '$(LDFLAGS)' antrea.io/theia/plugins/...
```

Comment:
Currently, we don't have and don't have a plan to add codes under cmd/ or pkg/, so I didn't add that directory.

Reply:
Got it. I thought we have cmd and pkg because I saw them in line 77 `$(GO) test -race -covermode=atomic -cover antrea.io/theia/cmd/... antrea.io/theia/pkg/... antrea.io/theia/plugins/..`. Maybe we should remove them from line 77/

Reply:
kk. Updated.